src/bodyparser_middleware.ts

```diff
@@ -140,6 +140,13 @@ export class BodyParserMiddleware {
     if (this.#isType(ctx.request, multipartConfig.types)) {
       debug('detected multipart request "%s:%s"', requestMethod, requestUrl)
 
+      if (!multipartConfig.enabled) {
+        throw new Exception('request content-type not supported', {
+          status: 415,
+          code: 'E_REQUEST_UNSUPPORTED_MEDIA_TYPE',
+        })
```

Comment:
Wouldn't it be better to create a custom exception class for this and expose it via an `errors` export, like the `@adonisjs/auth` package?
That way, people can use `instanceof errors....` in an error handler.

However, this would require new documentation for the new exception type...

Reply:
I followed the pattern already in use here: https://github.com/adonisjs/bodyparser/blob/develop/src/bodyparser_middleware.ts#L72-L89

Reply:
Oh, I see! Thank you for clarifying!